applications/nrf5340_audio/src/modules/sd_card_playback.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef _SD_CARD_PLAYBACK_H_
+#define _SD_CARD_PLAYBACK_H_
+
+#include <zephyr/kernel.h>
+
+/**
+ * @brief	Figure out whether or not the sd_card_playback module is active.
+ *
+ * @retval	true    Active.
+ * @retval	false   Not active.
+ */
+bool sd_card_playback_is_active(void);
+
+/**
+ * @brief	Play audio from .wav file from SD card. Only support for mono files.
+ *
+ * @note	Only support for 48k mono files
+ *
+ * @param[in]	filename	Name of file to be played. Path from root of SD card is accepted.
+ *
+ * @retval	0       Success.
+ * @retval	-EACCES SW codec is not initialized.
+ */
+int sd_card_playback_wav(char *filename);
+
+/**
+ * @brief	Play audio from LC3 file from SD card.
+ *
+ * @note	Only support for mono
+ *
+ * @param[in]	filename	Name of file to be played. Path from root of SD card is accepted.
+ *
+ * @retval	0       Success.
+ * @retval	-EACCES SW codec is not initialized.
+ */
+int sd_card_playback_lc3(char *filename);
+
+/**
+ * @brief	Mix pcm data from sd_card_playback module with audio stream out.
```

Comment:
This line is causing the doc build issue:

![image](https://github.com/nrfconnect/sdk-nrf/assets/41291385/75e76a5d-e7c6-4dda-bd7d-3c3c8752c671)